packages/bootstrap/scss/menu-button/_index.scss

```diff
@@ -2,6 +2,18 @@
 @import "../utils/_index.scss";
 
 
+// Module meta
+$_kendo-module-meta: (
+    name: "menu",
```

Comment:
name should be menu-button and dependencies should be corrected to those of the menu-button